packages/story-editor/src/components/canvas/karma/canvasKeys.karma.js

```diff
@@ -124,6 +124,9 @@ describe('Canvas Keyboard Shortcuts', () => {
     await fixture.events.click(
       fixture.screen.getByRole('option', { name: /^"Fade In" Effect$/ })
     );
+
+    // Synthetic delay to make the dropdown behave correctly
+    await new Promise((r) => setTimeout(r, 200));
```

Comment:
Nit:

```suggestion
    await fixture.events.sleep(200);
```

Source: https://github.com/GoogleForCreators/web-stories-wp/blob/39c9a737cba2174d8a2cac29da8075c7c876f9bd/packages/karma-fixture/src/events.js#L61-L69